lang/perl/perlmod.mk

```diff
@@ -34,9 +34,11 @@ PERLMOD_TESTSDIR:=/usr/share/perl/perlmod-tests
 
 define perlmod/host/relink
 	rm -f $(1)/Makefile.aperl
-	$(MAKE) -C $(1) perl
-	$(INSTALL_BIN) $(1)/perl $(PERL_CMD)
-	$(INSTALL_BIN) $(1)/perl $(STAGING_DIR_HOSTPKG)/usr/bin/perl
+	( flock -w 300 9 || { echo perlmod/host/relink: failed to acquire lock; exit 1; }; \
```

Comment:
Wonder if $(STAGING_DIR_HOST)/flock should be used instead.

Reply:
OK, I've made the change.